.gitignore

```diff
@@ -152,3 +152,5 @@ transmuxed_video
 
 # ngrok
 .ngrok_url
+
+*.jpg
```

Comment:
don't do it like this! it could hide some images that we want to commit.
Choose a directory name in which you are creaving images that you don't want to commit and add it to .gitignore. Maybe `/data/images`?